pkg/tools/log/defaultlogger.go

```diff
@@ -104,6 +104,8 @@ func (l *defaultLogger) WithField(key, value interface{}) Logger {
 	}
 }
 
+func (l *defaultLogger) SetLogLevel(level string) {}
```

Comment:
It doesn't look like valid log level handling :)
Please add some checks and returns in corresponding `Info`, `Warn`, ... methods.